internal/local/command_kafka_cluster.go

```diff
@@ -0,0 +1,16 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func (c *Command) newKafkaClusterCommand() *cobra.Command {
```

Comment:
Any reason why `Command` is public?
```suggestion
func (c *command) newKafkaClusterCommand() *cobra.Command {
```

Reply:
Good catch! It was not necessary